### PR TITLE
[styled-system]: Add missing justifyItems definition.

### PR DESCRIPTION
**Pull request by threehams**
JustifyItemsProps existed, but the export for justifyItems was missing.

Note that I added this under Grid instead of Flex because this property is ignore in Flexbox: [MDN: justify-items](https://developer.mozilla.org/en-US/docs/Web/CSS/justify-items).

Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/jxnblk/styled-system/blob/9358dff509fec3fbacbf8ff6e9a4be27420ebcbd/docs/table.md#flexbox
- [x] Increase the version number in the header if appropriate.
- [ ] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

### CODE REVIEW

File: types/styled-system/styled-system-tests.tsx [hunk 1]
```diff
@@ -137,8 +139,8 @@ const breakpointsPx = breakpoints.map(px);
 const mediaQueries = breakpoints.map(createMediaQuery);
 
 const boxStyle = variant({
-    prop: 'boxStyle',
-    key: 'box',
+    prop: "boxStyle",
```
Comment (threehams) [inline types/styled-system/styled-system-tests.tsx]: hm, looks like `prettier` was set to format on save globally. I can revert this, but changes are minimal and this does fix the split tab/spaces and single/double quotes within the file. Reviewer's choice.

File: types/styled-system/styled-system-tests.tsx [hunk 2]
```diff
@@ -285,6 +288,7 @@ const Grid: React.ComponentType<GridComponentProps> = styled`
     ${gridAutoColumns};
     ${gridTemplateRows};
     ${gridTemplateColumns};
+    ${justifyItems};
```
Comment (adam187) [inline types/styled-system/styled-system-tests.tsx]: This should probably be added to [`Flex`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/247a03b2198d698e7283095f951aebf549705668/types/styled-system/styled-system-tests.tsx#L259) not in `Grid`

**Comment by adam187** [comment]
Hey @threehams, thx for PR :) Please check out my comments.

File: types/styled-system/styled-system-tests.tsx [hunk 3]
```diff
@@ -273,7 +275,8 @@ interface GridComponentProps
         GridAutoColumnsProps,
         GridAutoRowsProps,
         GridTemplatesRowsProps,
-        GridTemplatesColumnsProps {}
+        GridTemplatesColumnsProps,
+        JustifyItemsProps {}
```
Comment (adam187) [inline types/styled-system/styled-system-tests.tsx]: This should probably be added to [`FlexComponentProps`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/247a03b2198d698e7283095f951aebf549705668/types/styled-system/styled-system-tests.tsx#L252)

**Comment by adam187** [comment]
@threehams Ok, I saw your comment why you added it to `Grid`. 

Problem is that in `styled-system` docs it's declared as `Flexbox` style function.
https://github.com/jxnblk/styled-system/blob/9358dff509fec3fbacbf8ff6e9a4be27420ebcbd/docs/table.md#flexbox

So i would still move it to `Flexbox` for constancy with `styled-system` docs for now.

You can still use it in your code where ever you like.

**Comment by threehams** [comment]
Sure, I can keep consistency with the library. It's a pretty confusing property regardless of where it goes.

**Comment by alloy** [comment]
I think the types should follow the actual upstream implementation, so probably best to merge as-is (after fixing the CI failure) and then bring up the CSS issue upstream.

File: types/styled-system/styled-system-tests.tsx [hunk 4]
```diff
@@ -273,7 +277,7 @@ interface GridComponentProps
         GridAutoColumnsProps,
         GridAutoRowsProps,
         GridTemplatesRowsProps,
-        GridTemplatesColumnsProps {}
+        GridTemplatesColumnsProps{}
```
Comment (adam187) [inline types/styled-system/styled-system-tests.tsx]: @threehams Revert this change please

**Review by adam187 — APPROVED** [review]

**Comment by alloy** [comment]
Thanks!
